stipendien/goingabroad.md

```diff
@@ -0,0 +1,119 @@
+
+Nach Deinem Forschungsprojekt hast Du die Möglichkeit weitere 30 Tage in den USA zu verbringen.
+In dieser Zeit steht es Dir frei das Land auf eigene Kosten zu erkunden.
+Unter Absprache mit den Förderern kannst Du Dir den Ort des Rückflugs selbst aussuchen.
+
+#### Mit welchem Themengebiet beschäftigst Du dich?
+Das Programm findet im Bereich der Hochenergiephysik statt.
+
+Im Wesentlichen beschäftigst Du Dich dann mit diesem Themenbereich:
+
+"High energy physics summer research projects for UG students at Purdue University are in the context
+of the analysis of the data recorded with the Compact Muon Solenoid (CMS) detector at the LHC. We are
+also deeply involved in the R&D for a new silicon pixel detector, which is currently developed for the
+high-luminosity upgrade of the LHC (HL-LHC).
+
+The Purdue group is involved in the measurement of the top quark coupling to the Higgs boson
+(Yukawa coupling) and in high precision measurements of top quark properties. Summer research projects
+for UG students will be tied to the group’s data analysis efforts. The Higgs boson gives mass to all
+elementary particles and the value of the top quark Yukawa coupling sheds light on the electroweak
+symmetry breaking process and even on the question whether the Universe is in a stable, meta-stable or
+instable vacuum state. Together with the precision measurement of top quark properties results can pin
+point to beyond the standard model contributions, which is what all particle physicists are searching
+for since decades. The data analysis heavily relies on methods that include topological multi-variate
+analysis, regularized matrix unfolding and complementary profiling techniques.
+
+There is also the opportunity to contribute in the R&D for the high-luminosity upgrade of the CMS
+detector following the HL-LHC upgrade to provide unprecedented instantaneous luminosities at a hadron
+collider. The Purdue Silicon Detector Laboratory (PSDL) is an in-house facility providing 2 large clean
+rooms in order to assemble highly integrated electronic circuits via a robotic gantry and test fully
+assembled silicon pixel modules with high precision probe stations. The PSDL produced about 900 silicon
+detector modules for the CMS forward silicon pixel detector currently commissioned at CERN/CMS. The group
+is also focused on the production of support structure prototypes from carbon foam, custom carbon fiber
+face sheets and mock-up silicon pixel modules. These silicon detector prototypes are tested for their
+thermal performance under the same conditions as later in the CMS detector. We use the existing PSDL
+capabilities for the mechanical and thermal studies, which include a curing oven and a cold box setup
+for thermal studies employing liquid CO2 coolant."
+
+Weitere Informationen zu Jung und seinem Team erhältst Du [hier](http://www.physics.purdue.edu/~jung196/index.html)
+
+#### Kann ich den Aufenthalt mit meiner Bachelorarbeit verbinden?
```

Comment:
Wenn die Bachelorarbeit verpflichtend Teil des Austauschprogramms ist, sollte das nicht hier stehen sondern in  der Überschrift! Das ist dann die zentrale Eigenschaft des Austausches und reduziert automatisch alle möglichen Teilnehmer auf Menschen im 6. Semester. Das muss von Anfang an klar sein. "Going Abroad – Bachelorarbeit in Purdue".